### PR TITLE
CI Lint + Fixes

### PR DESCRIPTION
**Pull request by leonardoventurini**
- Add lint workflow to GitHub actions (https://github.com/monti-apm/monti-apm-agent/blob/3a25685e9927c85aad369d885675e587bf81eda7/.github/workflows/lint.yml)
- Fix all linting errors or suppress them where it is too much trouble/risky.
- Fine tune eslint config a little bit

### CODE REVIEW

**Comment by afrokick** [comment]
@zodern is it ok to merge this pr? Or should it be divided into small PRs? I have #89 & #90 But if we merge this one I will close mine.

**Comment by leonardoventurini** [comment]
Sorry for making such a huge PR, but it was one of options presented in the task... and I believe we have something bigger in our hands related to fibers, and I wanted to get this done with completely to avoid tech debt.

File: lib/tracer/tracer_store.js [hunk 1]
```diff
@@ -2,7 +2,9 @@ import { _ } from 'meteor/underscore';
 import { EJSON } from 'meteor/ejson';
 let logger = Npm.require('debug')('kadira:ts');
 
-TracerStore = function TracerStore (options) {
+export let TracerStore;
+
+TracerStore = function (options) {
```
Comment (zodern) [inline lib/tracer/tracer_store.js:7]: Could these lines be combined for consistency with how it is done in other files?

File: lib/hijack/wrap_webapp.js [hunk 2]
```diff
@@ -44,14 +44,14 @@ export function checkHandlersInFiber () {
...
-const InfoSymbol = Symbol();
+const InfoSymbol = Symbol('InfoSymbol');
 
 export async function wrapWebApp () {
   if (!checkHandlersInFiber() || !canWrapStaticHandler) {
     return;
   }
 
-  const parseUrl = require('parseurl');
+  const parseUrl = Npm.require('parseurl');
```
[the remark this answers is not in the document]
Comment (leonardoventurini) [reply lib/hijack/wrap_webapp.js:54]: Isn't `Npm.require` the proper way to import packages which are declared in the `package.js`? Or should I disable the `require` rule?

File: .eslintrc.yml [hunk 3]
```diff
@@ -60,7 +65,7 @@ rules:
     - 2
     - before: true
       after: false
-  guard-for-in: 1
+  guard-for-in: 0
```
[the remark this answers is not in the document]
Comment (leonardoventurini) [reply .eslintrc.yml:68]: I don't know, it seemed useless to me, never saw an issue originate from that, unless the code is not properly written. I can add the guards to all the loops.

File: lib/models/system.js [hunk 4]
```diff
@@ -6,11 +6,12 @@ import { getFiberMetrics, resetFiberMetrics } from '../hijack/async.js';
 import { getMongoDriverStats, resetMongoDriverStats } from '../hijack/mongo_driver_events.js';
 import { KadiraModel } from './0model';
 import { EventLoopMonitor } from '../event_loop_monitor.js';
+import { Ntp } from '../ntp';
 
 export function SystemModel () {
   this.startTime = Ntp._now();
   this.newSessions = 0;
-  this.sessionTimeout = 1000 * 60 * 30; // 30 min
+  this.sessionTimeout = 1000 * 60 * 30;
```
[the remark this answers is not in the document]
Comment (leonardoventurini) [reply lib/models/system.js:14]: Well, it looked obvious enough for me, adding it back

File: lib/hijack/wrap_webapp.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (zodern) [reply lib/hijack/wrap_webapp.js]: `require` works for importing them too. I am in favor of disabling the require rule.

**Comment by zodern** [comment]
Thanks @leonardoventurini

**Pull request merged**